yt_dlp/extractor/aflwomens.py

```diff
@@ -0,0 +1,60 @@
+# coding: utf-8
+import re
+
+from .common import InfoExtractor
+
+from ..utils import urljoin
+
+
+class AFLWomensPlaylistIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?womens\.afl/(?P<id>match-(?:replays|highlights)|video/[a-z0-9-]+)$'
+    _TESTS = [{
+        'url': 'https://www.womens.afl/match-replays',
+        'playlist_mincount': 3,
+        'info_dict': {
+            'id': 'match-replays',
+            'title': 'Match Replays',
+            'entries': [{
+                'url': 'https://www.womens.afl/video/86436/aflw-match-replay-west-coast-v-richmond',
+                '_type': 'url'
+            }, {
+                'url': 'https://www.womens.afl/video/86276/r7-carlton-v-st-kilda',
+                '_type': 'url'
+            }, {
+                'url': 'https://www.womens.afl/video/86190/aflw-match-replay-gws-v-adelaide',
+                '_type': 'url'
+            }],
+        },
```

Comment:
```py
        'info_dict': {
            'id': 'match-replays',
            'title': 'Match Replays',
        }, 'playlist': [{
            'md5': 'xxx',
            'info_dict': {
                ...
            },
        }, ...]
```
this is how entries should be tested

Reply:
That presumably still won't fix the issue of the playlists changing, though?

Reply:
Is there no fixed playlists to test against? You can limit the test's scope to certain items like this if needed: `'params': {'playlistitems': '1-5'}`